src/Ranking.elm

```diff
@@ -1,17 +1,105 @@
 module Ranking exposing (Model, Msg, init, update, view)
 
-import Debug exposing (todo)
-import Zipper exposing (Zipper)
+import Browser
+import List.Zipper exposing (current, after)
+import SetZipper exposing (crateZipper)
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (centerX, centerY, column, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Maybe
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view >> layout []
+        , update = update
+        }
 
 
 type alias Model =
     { teams : Zipper (Zipper Team)
-    , isShowingInfo : Bool
+    , isShowingInfo : Bool -- What I need to do with this variable?
+    , inPage : Int
+    , windowHeight : Int
     }
 
 
+type Msg
+    = NextPage
+
+
 type alias Team =
     { number : Int
     , name : String
     , score : Float
     }
+
+
+moveZipperBy : Int -> Zipper a -> Zipper a -- Never failed
+moveZipperBy moveBy zipper =
+    let
+        zipperNextCurrent : Maybe (Zipper a)
+        zipperNextCurrent = next zipper
+    in
+        if moveBy == 0 || zipperNextCurrent == Nothing then
+            zipper
+        else
+            next zipper
+            moveZipperBy (moveBy - 1) zipper
+
+
+getNeededList : Int -> Zipper a -> List a
+getNeededList neededInPage zipper =
```

Comment:
I would suggest thinking on a more generic name for function and variable. Think about what the function does rather than when you use it in this program context.

Reply:
Still applies...